demo/Collapsible.demo.js

```diff
@@ -5,49 +5,41 @@ const CollapsibleDemo = () => (
   <Fragment>
     <code>
       <pre>
-        {'<Collapsible />'}
-        <br />
-        Props:
-        <br />
-        openOnInit: true
-        <br />
-        transTime: undefined
-        <br />
-        transCurve: undefined
+        {'<Collapsible'}
+        <ul style={{ listStyle: 'none' }}>
+          <li>
+            {'openOnInit={true}'}
+          </li>
+        </ul>
+        {'/>'}
       </pre>
     </code>
     <Collapsible
       classPrefix="custom"
       clickableNode={<span>click me</span>}
       openOnInit
     >
-      <span>to collapse content</span>
+      <span>beep bop robots</span>
```

Comment:
Let's use a snippet of lorem ipsum for everything in these demos.

Here's an example of length: `Modnam llam leosed magnis mauris iennam ibulum`

Be sure that each is unique.